ee/ucp/kubernetes/storage/use-azure-files.md

```diff
@@ -238,6 +240,48 @@ spec:
 EOF
 ```
 
+### Troubleshooting
+
+When creating Persistent Volume Claims the volume may constantly stay in a
```

Comment:
Suggestion: "When creating Persistent Volume Claims, the volume may constantly stay in a..."